test/topology/util.py

```diff
@@ -234,6 +235,48 @@ async def check_system_topology_and_cdc_generations_v3_consistency(manager: Mana
         # Check that the contents fetched from the current host are the same as for other nodes
         assert topo_results[0] == topo_res
 
+async def start_writes_to_cdc_table(cql: Session, concurrency: int = 3):
+    logger.info(f"Starting to asynchronously write, concurrency = {concurrency}")
+
+    stop_event = asyncio.Event()
+
+    ks_name = unique_name()
+    await cql.run_async(f"CREATE KEYSPACE {ks_name} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 3}} AND tablets = {{ 'enabled': false }}")
+    await cql.run_async(f"CREATE TABLE {ks_name}.tbl (pk int PRIMARY KEY, v int) WITH cdc = {{'enabled':true}}")
+
+    stmt = cql.prepare(f"INSERT INTO {ks_name}.tbl (pk, v) VALUES (?, 0)")
+    stmt.consistency_level = ConsistencyLevel.ONE
+
+    async def do_writes():
+        iteration = 0
+        while not stop_event.is_set():
+            try:
+                await cql.run_async(stmt, [iteration])
+            except Exception as e:
+                logger.warning(f"Write failed: {e}")
+            iteration += 1
+            await asyncio.sleep(0.01)
```

Comment:
Why sleep here? Is there a reason why we want to reduce the number of writes?

Reply:
> Why sleep here? Is there a reason why we want to reduce the number of writes?

To reduce CPU usage. There are already three fibers doing concurrent writes

Reply:
Yes, it just seemed a bit strange to me that you increase the number of writes by introducing 3 fibers and then you decrease it with sleeps. But what we want concurrency here, so it's fine.

Also, since you want to reuse it in the recovery test, it's even better. Spamming writes in tests with node loss can be problematic. For example, background writes can use up all memory and block next writes, which can slow down a test or even be flaky.